components/FlashCard.js

```diff
@@ -48,7 +48,7 @@ export default function FlashCard({
   deleteCard,
 }) {
   const [showAnswer, setShowAnswer] = useState(false);
-  const [isClicked, setIsClicked] = useState(false);
+  const [isDeleteOptionVisible, setIsDeleteOptionVisible] = useState(true);
```

Comment:
If you want to keep the initial state value as `true` to avoid negation in the ternary operator, you could rename the state as follows to make the logic more intuitive:

```suggestion
const [showDeleteButton, setShowDeleteButton] = useState(true);
```